### DISCUSSION TITLE
RAWTherapee on Windows10 Migration from RT5.8 to RT5.9

### DISCUSSION THREAD

**Discussion by ALOEILALAIN** [Beep6581/RawTherapee#6669] [Q&A] [answered]
The .pp3 profiles created on RT5.8 do not give the same result (JPEG) on RT5.9 (especially for negative films but also for digital)
How can I solve this problem? 
Should I use the 2 versions in parallel and recreate all the .pp3 profiles?
If yes, it doesn't work because preferences and profiles are stored in the cache and are not reset at startup 
 of RT5.8 or RT5.9 from C:\Users\myuser\AppData\Local\RawTherapee\profiles\5.8 or C:\Users\myuser\AppData\Local\RawTherapee\profiles\5.9
Should we use two different users? Or do you have a better solution?

Another question:
Where can I find the new description RT5.9 of color - film negative ?

**Comment by rom9** [comment, marked as answer]
Hello,
 the film negative tool should be backwards compatible and convert the profile automatically. Please, can you open an issue about this, possibly uploading a sample .pp3 file showing the problem?
If you could also share an image showing the problem, it would be great.
Thanks :-)

**Reply by ALOEILALAIN** [reply]
Hello,
The negative film tool does not seem to be the problem but I transferred a negative image RT5_8_IMG_EOS M51849R.jpg and a digital image RT5_8_IMG_EOS 200D0861R.jpg generated
originally with RT5.8 and the same two images generated with the new version RT5.9. See below
Unfortunately I can't transfer the *.pp3 files because this extension is not accepted,
You can see the difference between RT5.8 and RT5.9 JPEGs
Best regards

**Comment by ALOEILALAIN** [comment]
![RT5_8_IMG_EOS 200D0861R](https://user-images.githubusercontent.com/123459062/214395036-165ecf0b-5858-4e68-88ad-975fb8216712.jpg)
![RT5_8_IMG_EOS M51849R](https://user-images.githubusercontent.com/123459062/214395192-7909d009-5a6d-4aba-a782-16b89df58f9d.jpg)
![RT5_9_IMG_EOS 200D0861R](https://user-images.githubusercontent.com/123459062/214395778-13c68aa8-1c1b-459b-bc29-a23b7cb77a6d.jpg)
![RT5_9_IMG_EOS M51849R](https://user-images.githubusercontent.com/123459062/214395789-045d1ac3-faf8-4086-902a-dee82a122a9f.jpg)

**Comment by Lawrence37** [comment]
This looks similar #6639. @ALOEILALAIN can you try the dev version to see if the output matches 5.8?
https://github.com/Beep6581/RawTherapee/releases/tag/nightly-github-actions

**Reply by ALOEILALAIN** [reply]
Hello,
I installed RT 5.9-58-g7afdfc1e2 but when I start it, the Libsharpyuv-0.ddl library is missing. What can I do?

**Reply by Lawrence37** [reply]
Once #6671 is closed, the new builds should run without the error.

**Reply by Lawrence37** [reply]
The DLL problem has been fixed. Try the latest dev build.

**Comment by ALOEILALAIN** [comment]
Hello,

The new version: 5.9-61-gb1490e24c works.
But the JPEGs generated with RT5.9-dev are absolutely identical to RT5.9 and still different from RT5.8 (the problem is pending see above)
I attach the PP3.txt files as additional information. (remove the .txt)

Thanks :-)


[IMG_EOS 200D0861.CR2.pp3.txt](https://github.com/Beep6581/RawTherapee/files/10555782/IMG_EOS.200D0861.CR2.pp3.txt)
[IMG_EOS M51849.CR2.pp3.txt](https://github.com/Beep6581/RawTherapee/files/10555783/IMG_EOS.M51849.CR2.pp3.txt)

**Reply by rom9** [reply]
Ok, so i'm afraid this is a problem with the processing profile upgrade :-(
Please, in order to easily reproduce the issue, could you also share one of the RAW images, along with the related .pp3 file ?
Not necessarily one of the pictures above; any image affected by this problem, as long as it's in RAW format.
Thanks
